tests/snmpsim/fortigate_sdwan.snmprec

```diff
@@ -98,16 +98,3 @@
 1.3.6.1.4.1.12356.101.4.9.2.1.14.4|4|PRIVATE-B
 1.3.6.1.4.1.12356.101.4.9.2.1.14.5|4|wan1
 1.3.6.1.4.1.12356.101.4.9.2.1.14.6|4|wan2
-1.3.6.1.4.1.12356.101.12.1.1.0|2|2
-1.3.6.1.4.1.12356.101.12.2.2.1.2.1.1|4|PRIVATE-A
```

Comment:
Any reason for deleting test data?

Reply:
> Any reason for deleting test data?

It's unrelated to the test. It references SSL-VPN related statistics and has nothing to do with the SD-WAN statistics for which it is used. If something changes in the SSL-VPN snmp data the test errors for the wrong reason.